File: pyg_manager_web/src/main/webapp/js/controller/loginController.js
app.controller("loginController", function ($scope,$controller,loginService) {
    // $controller('baseController', {scope: $scope});

    //获取当前用户的登录名
    $scope.getLoginName = function () {
        loginService.getLoginName().success(function (data) {
            $scope.loginName = data.loginName;
        });
    };

});

File: pyg_shop_web/src/main/java/com/songxin/pyg/shop/controller/LoginController.java
package com.songxin.pyg.shop.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

/**
 * @author fishsx
 * @date 2018/12/11 下午9:06
 */
@RestController
@RequestMapping("login")
public class LoginController {

    @RequestMapping("getLoginName")
    public Map getLoginName() {
        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        Map<String, Object> map = new HashMap<>();
        map.put("loginName", name);
        return map;
    }
}
